### PR TITLE
Exclude complexes from the set of incomplete elements in the task list

### PR DESCRIPTION
**Pull request by maxkfranz**
Ref : Complexes are erroneously shown as incomplete by submit UI #670

### CODE REVIEW

File: src/client/components/tasks.js [hunk 1]
```diff
@@ -93,7 +94,7 @@ class TaskView extends DataComponent {
     let { document, bus } = this.props;
     let done = document.submitted() || document.published();
     let incompleteEles = this.props.document.elements().filter(ele => {
-      return !ele.completed() && !ele.isInteraction();
+      return !ele.completed() && !ele.isInteraction() && !ele.type() === ENTITY_TYPE.COMPLEX;
```
Comment (jvwong) [inline src/client/components/tasks.js]:
```js
return !ele.completed() && !ele.isInteraction() && !(ele.type() === ENTITY_TYPE.COMPLEX);
```

File: src/client/components/tasks.js [hunk 2]
```diff
@@ -93,7 +94,7 @@ class TaskView extends DataComponent {
...
     });
 
     let ntfns = document.entities().filter(ele => !ele.completed()).map(ele => {
```
Comment (jvwong) [inline src/client/components/tasks.js]: This needs to filter out the `ele`s that are type `complex` as well.

Comment (maxkfranz) [reply src/client/components/tasks.js]: Isn't that just the same as `ENTITY_TYPE.COMPLEX`?

File: src/client/components/tasks.js (hunk 1 above)
Comment (maxkfranz) [inline src/client/components/tasks.js]:
```suggestion
      return !ele.completed() && !ele.isInteraction() && ele.type() !== ENTITY_TYPE.COMPLEX;
```

File: src/client/components/tasks.js (hunk 2 above)
Comment (maxkfranz) [reply src/client/components/tasks.js]: According to this condition, any complex is always excluded from the incomplete element list.

Comment (jvwong) [reply src/client/components/tasks.js]: I'm referring to the `ntfns` - shouldn't this filter out complexes like in `incompleteEles`? Right now in the UI I still see the complexes as being listed as incomplete.

Comment (jvwong) [reply src/client/components/tasks.js]: image

Comment (maxkfranz) [reply src/client/components/tasks.js]: That issue is only reproducible with the extra node in the top-left of your screenshot.  With the change, this case look OK:

Screen Shot 2020-03-20 at 11 28 05 AM

**Pull request merged**